proto/attest.proto

```diff
@@ -125,10 +125,15 @@ message ContainerState {
   string image_reference = 1;
   bytes image_digest = 2;
   RestartPolicy restart_policy = 3;
+  string image_id = 4;
+  repeated string args = 5;
+  map<string, string> env_vars = 6;
 }
 
 message AttestedCosState {
   ContainerState container = 1;
+  int32 cos_version = 2;
+  int32 launcher_version = 3;
```

Comment:
Why int32 instead of an unsigned number, or a major/minor helper struct?

Reply:
Good catch on the signed. Do we expect to have major/minor versions? What will claims look like?